modyn/config/schema/pipeline.py

```diff
@@ -480,16 +534,6 @@ class EvaluationConfig(BaseModel):
         min_length=1,
     )
 
-    # Additional validation
-
-    @model_validator(mode="after")
```

Comment:
Thanks for improving the pipeline model! 🥇 

Why do we want to drop the `dataset_id` uniqueness constraint though?

Reply:
I didn't find a working solution for the moment. The `model_validator` does not work as I checked, but I cannot find a correct one.